### DISCUSSION TITLE
Filament assets triggering CORS error when in production

### DISCUSSION THREAD

**Discussion by faridsa** [filamentphp/filament#6679] [Help]
I have a vanilla laravel 10 + Filament 2 installation, and in local environment everything works fine.

The point is when deploying to web server, I can't access to filament CSS because a CORS error in firefox

```
GET https://biosystems.devtesting.com.ar/admin [HTTP/2 200 OK 832ms]
GET http://[::1]:5173/@vite/client
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://[::1]:5173/@vite/client
Module source URI is not allowed in this document: “http://[::1]:5173/@vite/client”.
GET http://[::1]:5173/resources/css/app.css
```

Safari gives a different error but the same result. No styles loaded
![Screen Shot 2023-06-02 at 16 02 16](https://github.com/filamentphp/filament/assets/23212965/5949b41a-bac2-41a2-b1e2-d362d225cbf2)

And in chrome I got
```
Failed to load resource: net::ERR_CONNECTION_REFUSED client:1 
Failed to load resource: net::ERR_CONNECTION_REFUSED  app.css:1    
```

And clicking con filename returns: **No resource with given url found** 

In my .env file I have:
```
APP_NAME=BioSystems
APP_ENV=production
APP_KEY=base64:J62nECBoybrAcV8qwjLbAzKnw7bGNnPYNbFS8DTvHQo=
APP_DEBUG=false
APP_URL=https://biosystems.devtesting.com.ar
```

My SSL is working fine

![Screen Shot 2023-06-02 at 15 59 39](https://github.com/filamentphp/filament/assets/23212965/17e61656-b86c-4f7d-837e-7f9a4fdf9d01)

Any suggestion will be appreciated.

Regards,

**Comment by danharrin** [comment]
Make sure you run `npm run build` instead of `npm run dev` in production. This is not a Filament problem.

**Reply by faridsa** [reply]
Yes, npm build generates `/public/assets/app-a61bfa33.css` (145.311b) and `/public/assets/app-0d91dc04.js` (29.034b) but none of those are loaded by admin page.
I don't understand why you say it has nothing to do with filament when I'm using nothing but filament. Can you expand your idea a bit?
Thank you so much!

**Reply by danharrin** [reply]
The URLs that Vite generates (like @vite/client) are not controlled by Filament. I don't think it should be loading that in production?

**Reply by faridsa** [reply]
You were right. There was a file with name `hot` in my public folder which is calling `http://[::1]:4000` and thus prevent loading assets from build. I've deleted it and everything works fine now!
Thank you!
